src/mz/tests/e2e.rs

```diff
@@ -617,6 +617,7 @@ mod tests {
             status: &'a str,
         }
 
+        // TODO: add aws/us-west-2 once this test is fixed/re-enabled
```

Comment:
Should we add `us-west-2` in this PR, since we're not merging this till launch anyway?

Reply:
(I have another PR to remove this e2e test dead code.)

Reply:
I will create another PR to start testing `aws/us-west-2`. This comment can be removed (that part is deadcode.)

Reply:
We should change [this variable](https://github.com/MaterializeInc/materialize/blob/aa8671d83f6c3262c417cfefde27859b4216fee1/test/mz-e2e/mzcompose.py#L29) and make [this assertion](https://github.com/MaterializeInc/materialize/blob/aa8671d83f6c3262c417cfefde27859b4216fee1/test/mz-e2e/mzcompose.py#L211) work.

The `TODO` comment is in an old e2e test that is now deadcode (I have another PR for removing it.)